faq reindexing-typos (PR from TinaCMS) (#4002)
tinacms/tina.io: content/docs/introduction/faq.mdx

2. Verify that the correct branch has been specified in the config properties passed to defineConfig in `tina/config.ts`. Note, that this value may be set as an environment variable in your CI build process
3. Verify that the branch exists in the GitHub repository
4. Pull down the branch into your TinaCloud project. <Youtube embedSrc="https://youtube.com/embed/GDAQle7Q-sY" /> **Video: TinaCloud - How to index missing branches (3 min)**

> 5. You can also force a branch into your TinaCloud project by making a whitespace change to the `tina/tina-lock.json` file in that branch, committing it, and pushing it to GitHub. This will initiate indexing for the branch, and (after a few minutes) the error should be resolved

Reviewer: ```suggestion
5. You can also force a branch to be reindexed by TinaCloud by making a whitespace change to the `tina/tina-lock.json` file in that branch, committing it, and pushing it to GitHub. This will initiate indexing for the branch, and (after a few minutes) the error should be resolved
```
